app/grandchallenge/challenges/models.py

```diff
@@ -707,7 +725,7 @@ def should_be_open_but_is_over_budget(self):
             for phase in self.phase_set.all()
         )
 
-    @cached_property
+    @property
```

Comment:
I think it might be good to keep this as `approved_compute_costs_euro_millicents` involves a database query. To ensure the value you can send it to the task as an argument. But, we probably do not need such a level of detail in the email, just sending the threshold crossed would be okay.

Reply:
I think this is used in two templates, the challenge statistics page and the challenge cost overview. I'm not 100% sure but I think making this a normal property would mean recalculating this every time we search or filter or order the cost overview table, for example, which would not be desirable.

Reply:
Good points. Reverted this.